OloEditor/src/Panels/GamepadDebugPanel.cpp

```
@@ -9,7 +9,7 @@
 
 namespace OloEngine
 {
-    void GamepadDebugPanel::OnImGuiRender(bool* p_open)
+    void GamepadDebugPanel::OnImGuiRender(bool* p_open) const
```

Cppcheck (2.20.0)

[style] The function 'OnImGuiRender' is never used.

(unusedFunction)